src/components/DashboardNoTeam/ConfirmTeamCreation/ConfirmTeamCreation.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { Box, Text, Heading, Button, HStack, VStack } from '@chakra-ui/react';
+
+const ConfirmTeamCreation = props => {
+  const CREATE_TEAM = gql`
+    mutation createTeam($name: String!, $organization: String!) {
+      createTeam(name: $name, organization: $organization, amountRaised: 0, itemsSold: 0) {
+        id
+      }
+    }
+  `;
+
+  const ADD_USER_TO_TEAM = gql`
+    mutation updateUser($id: ID!, $teamId: String) {
+      updateUser(id: $id, teamId: $teamId) {
+        id
+      }
+    }
+  `;
```

Comment:
I suggest having all resolvers imported to make it easier for other devs to reuse the same queries/mutations

Reply:
agree with this, we are defining queries/mutations inline all over the code already though, will fix this at a later time